src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
I would really suggest to not do that, even
for the optional script. Allow it to have --list
so the user can see what's available, and allow
it to take the dos name from command-line.
If you want, you can also provide a separate
TUI script to wrap that process. This script will
be optional-optional. :)